symfony.lock

```diff
@@ -1,4 +1,61 @@
 {
+    "babdev/pagerfanta-bundle": {
```

Comment:
This needs to be removed from the diff. Only a subset of `symfony.lock` is added to this repo in order not to disable fetching latest recipes for each of the new projects.

The subset which is commited is related to keeping the structure of the project inline with ibexa, without Flex interfering with it.